quora-service/src/main/java/com/upgrad/quora/service/business/AnswerService.java

```diff
@@ -37,7 +41,78 @@ public AnswerEntity createAnswer(final UserAuthEntity userAuthTokenEntity, final
     answerEntity.setUser(userAuthTokenEntity.getUserEntity());
     answerEntity.setQuestion(questionEntity);
     return answerDao.createAnswer(answerEntity);
+  }
+
+  /**
+   *
+   * @param questionID - uuid of the question for which Fetch all Answers
+   * @return - List of AnswerEntity
+   * @throws AuthorizationFailedException - throwed if the user is not authorized to edit the question
+   * @throws InvalidQuestionException - throws exception if the Provided QuestionID Not Present in DB
+   */
+  public List<AnswerEntity> getAllAnswer(String questionID) throws AuthorizationFailedException, InvalidQuestionException {
+    QuestionEntity question = questionDao.getQuestion(questionID);
+    if(question == null)
+    {
+      throw new InvalidQuestionException("QUES-001","The question with entered uuid whose details are to be seen does not exist");
+    }
+    return answerDao.getAllAnswer(questionID);
+  }
+
+  /**
+   *
+   * @param answerEntity - AnswerEntity that to be edited
+   * @param userAuthToken - authorization header containing the accessToken
+   * @return - Deleted AnswerEntity
+   * @throws AuthorizationFailedException - throwed if the user is not authorized to edit the question
```

Comment:
I think this must be 
"throwed if user is not authorized to edit the answer"instead of question